examples/pruning/keras_integration.py

```diff
@@ -0,0 +1,94 @@
+"""
+Optuna example that demonstrates a pruner for Keras.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+Keras and MNIST, where the architecture of the neural network is optimized. Throughout the
+training of neural networks, a pruner observes intermediate results and stops unpromising trials.
+
+You can run this example as follows:
+    $ python keras_integration.py
+
+"""
+
+import optuna
+from optuna.integration import KerasPruningCallback
+
+import keras
+from keras.datasets import mnist
+from keras.layers import Dense
+from keras.models import Sequential
+from keras import optimizers
+from keras import regularizers
+
+
+BATCHSIZE = 128
+CLASSES = 10
+EPOCHS = 20
+
+
+def model_fn(trial):
+    # Model
```

Comment:
Could you explain the target hyperparameters as can be seen in [chainer example](https://github.com/pfnet/optuna/blob/master/examples/pruning/chainermn_integration.py#L36)?

Reply:
WIll do.

Reply:
Thank you for your quick action.

I have nits about comments.
Could you add a period at the end of this sentence?
And also, please fix the comment style in this example like
 [chainer example](https://github.com/pfnet/optuna/blob/master/examples/pruning/chainer_integration.py#L87).
Some comments lack periods and/or start with small letters.

Reply:
Okay. Fixed that.

Reply:
Alright.